internal/common/validation/validation.go

```diff
@@ -99,6 +101,37 @@ func ValidateModuleNamespace(namespace string) error {
 	return nil
 }
 
+func ValidateResources(resources contentprovider.ResourcesMap) error {
+	for name, link := range resources {
+		if name == "" {
+			return fmt.Errorf("%w: name must not be empty", commonerrors.ErrInvalidOption)
+		}
+
+		if link == "" {
```

Comment:
why not use `url.Parse` to handle empty link

Reply:
tbh, an empty link is not a valid URL quite correct to me😄 , but this up to you if you want to keep like this.

Reply:
yeah I would opt to keep it if you don't mind (not super visible in the github view but there is a double whitespace between `link` and `is` and this bothers me :D